PyOpenWorm/evidence.py

```diff
@@ -85,4 +85,37 @@ def identifier_augment(self):
         return self.make_identifier(s)
 
 
+def evidence_for(qctx, conn, Evidence, Context):
+    """
+    Returns an iterable of Evidence  
+    """
+    ctxs = query_context(conn.conf['rdf.graph'], qctx)
+    for c in ctxs:
+        mqctx = Context(conf=conn.conf)
+        print('CONTEXT', c.identifier)
+        ev = mqctx.stored(Evidence)()
+        ev.supports(Context(ident=c.identifier, conf=conn.conf).rdf_object)
+        ev_objs = []
```

Comment:
this shouldn't be reset each time. It's just a fluke that it was different in different versions of Python -- the context which had nothing attached just happened to come last in the tests that failed.

Reply:
Okay, got it - [Document, Document, None]